integration/images/volume-copy-up/Makefile

```diff
@@ -15,13 +15,75 @@
 all: build
 
 PROJ=gcr.io/k8s-cri-containerd
-VERSION=1.0
+VERSION=1.1
 IMAGE=$(PROJ)/volume-copy-up:$(VERSION)
 
-build:
-	docker build -t $(IMAGE) .
+# Architectures supported: amd64, arm, arm64, ppc64le and s390x
+ARCH ?= amd64
+# Operating systems supported: linux, windows
+OS ?= linux
+# OS Version for the Windows images: 1809, 1903, 1909 2004, 20H2
+OSVERSION ?= 1809 1903 1909 2004 20H2
```

Comment:
I don't think we will need 1903/1909. 1903 is already EOL and 1909 ends in May

Reply:
Done